specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/Jobs.json

```diff
@@ -625,6 +625,124 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/resume": {
+      "post": {
+        "tags": [
+          "Jobs"
+        ],
+        "summary": "Resumes a suspended job",
+        "operationId": "Jobs_Resume",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
```

Comment:
>/v3 [](http://example.com/codeflow?start=68&length=3)

use the v5 please

Reply:
done, thank you!

Reply:
hi @raosuhas ...for v5/, i tried updating the whole `jobs.json` file but got an error that the latest stable version conflicts with the intended changes

the run with the error: https://github.com/Azure/azure-rest-api-specs/pull/29788/checks?check_run_id=27634707415

i reverted back to v3 for all versions